dds/DCPS/NetworkConfigUpdater.h

```diff
@@ -8,11 +8,10 @@
 #ifndef OPENDDS_DCPS_NETWORKCONFIGUPDATER_H
 #define OPENDDS_DCPS_NETWORKCONFIGUPDATER_H
 
-#include "ace/config.h"
-
 #ifdef ACE_HAS_IOS 
 #define OPENDDS_NETWORK_CONFIG_UPDATER
 
+#include "ace/os_include/os_ifaddrs.h"
```

Comment:
There is a section in the developer guidelines about the order and style of includes.